Code/Tools/ProjectManager/Source/ToggleCheckbox.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QCheckbox>
+#endif
+
+namespace O3DE::ProjectManager
+{
+    class ToggleCheckbox
```

Comment:
Can we not use the default toggle switch instead of making our own class?
```
#include <QCheckBox>
#include <AzQtComponents/Components/Widgets/CheckBox.h>

QCheckBox* toggleSwitch;

// Assuming you've created a QCheckBox already (either in code or via .ui file), make it into a ToggleSwitch like this:
CheckBox::applyToggleSwitchStyle(toggleSwitch);
```

Reply:
Oh, that makes more sense, removed custom class, only thing is that checkbox looks a bit different than the one used in Project manager. It has a blue outline instead of a white one but it's fine.